test/usecases/run_test.sh

```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/env bash
+set -e
+
+nmodl="$1"
+usecase_dir="$2"
+
+pushd "${usecase_dir}"
+
+rm -r x86_64 tmp || true
+
+nrnivmodl -nmodl "${nmodl}"
+x86_64/special simulate.py
```

Comment:
as some people develop on apple-m1, would be nice to change it.

Reply:
If you have one please try, documentation suggests that it should be `uname -m`.

Reply:
I can confirm it works on the M1 Mac

Reply:
ty